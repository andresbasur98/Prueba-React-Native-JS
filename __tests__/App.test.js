import React from 'react'
import { SumissionItem } from "../src/components/SumissionItem";
import renderer from 'react-test-renderer';


test('Mostrar correctamente el componente', () => {
 
    const item = {
        index: 1,
        name: 'eer',
        surname: 'rkr',
        age: 3
    }

 const tree = renderer.create(<SumissionItem sumissionItem={item} />).toJSON();
 expect(tree).toMatchSnapshot();

})
