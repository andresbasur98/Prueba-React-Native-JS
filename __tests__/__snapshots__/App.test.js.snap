// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mostrar correctamente el componente 1`] = `
<View
  style={
    Object {
      "marginBottom": 10,
      "marginLeft": 10,
    }
  }
>
  <Text>
    Submission 
    1
  </Text>
  <View
    style={
      Object {
        "marginLeft": 10,
      }
    }
  >
    <Text>
      eer
    </Text>
    <Text>
      rkr
    </Text>
    <Text>
      3
    </Text>
  </View>
</View>
`;
